### DISCUSSION TITLE
Uno-Check : Xcode versioning confusion?

### DISCUSSION THREAD

**Discussion by baskren** [unoplatform/uno#18794] [General]
Just upgraded uno-check to `1.27.0`.  When I ran it, I got the following error:

```text
🔎 XCode 15F31d Checkup...
  ❌ Xcode.app (16.0 15F31d) not installed.
```

At that time, I had Xcode 16.1 and 15.4 installed on my machine ... so I installed Xcode 16.0.  Next, I run uno-check again and get the same error.

<img width="529" alt="image" src="https://github.com/user-attachments/assets/a614ce31-fb2c-4632-8ee6-d515cf4cb64d">

OK, I'm thinking maybe I'm pointing to the wrong version of the Xcode Command Line Tools?  So I launch Xcode and I notice the following...

<img width="529" alt="image" src="https://github.com/user-attachments/assets/e48d9cc7-609d-4d31-b548-334865d744df">

Hmmm.  The error is asking for `16.0 15F31d` but the options say:

- 16.0 : 16A242d
- 15.4 : 15F31d

I've tried both, checked the versions via command line (see below), and I still get the same error in uno-check.  

```text
ben@bc3-m2-mbp ~ % xcodebuild -version
Xcode 16.0
Build version 16A242d
ben@bc3-m2-mbp ~ % uno-check
...
🔎 XCode 15F31d Checkup...
  ❌ Xcode.app (16.0 15F31d) not installed.
...

```

```
ben@bc3-m2-mbp ~ % xcodebuild -version
Xcode 15.4
Build version 15F31d
ben@bc3-m2-mbp ~ % 
ben@bc3-m2-mbp ~ % uno-check
...
🔎 XCode 15F31d Checkup...
  ❌ Xcode.app (16.0 15F31d) not installed.
...
```

To date, I've had no issues using version 16.1 of the Xcode Command Line tools.  Is it safe to ignore this error?

**Comment by jeromelaban** [comment]
Yes, it was safe to ignore the error, and this is fixed in the latest build of uno.check, run `dotnet tool update -g uno.check` to get it.

**Reply by baskren** [reply]
Got it!  Thanks!
